vagrant/README.md

```diff
@@ -0,0 +1,88 @@
+# Setup Local Satellite Cluster
+
+The provided Vagrantfile and Ansible playbooks can be used to initialize a
+local satellite cluster and quickly test changes in development environment.
+The default configuration will setup a three node cluster 
+(k8s-master, node-1, node-2). Kubernetes, Serf, and Satellite will be installed
+on each node.
+
+## Dependencies
+*   [vagrant](https://www.vagrantup.com/)
+*   [ansible](https://www.ansible.com/)
+
+## Ansible
+
+*   The ansible directory contains a default ansible.cfg file and a few play books
+    used to initialize the local satellite cluster.
+
+    -   **credentials.yaml** initializes server & client credentials to be provide
+        authentication by satellite agents.
+
+    -   **k8s.yaml** initializes a kubernetes cluster with 1 master and 2 nodes.
+
+    -   **satellite.yaml** initializes satellite on each node and starts satellite
+        as a service.
+
+    -   **scripts.yaml** adds a few helpful scripts to run client commands.
+
+    -   **serf.yaml** installs and starts serf on each node.
+
+    -   **update-satellite.yaml** updates the satellite binary on each node and 
+        restarts the service.
+
+
+## Instructions
+
+Initialize virtual machines, install dependencies, and start satellite:
+```console
+$ make all
+```
+
+To run an individual playbook:
+```console
+$ make ansible-[playbook-name]
+```
+
+To update satellite binary:
+```console
+$ make ansible-update-satellite
+```
+
+To destroy cluster:
+```console
+$ make destroy
+```
+
+To ssh into a cluster (k8s-master, node-1, node-2):
+```console
+$ vagrant ssh k8s-master
+```
+
+## Scripts
+
+A few scripts are provided once ssh'd into a node.
+
+Get status:
+```console
+$ ./status.sh
+```
+
+Get history:
+```console
+$ ./history.sh
+```
+
+Drop packets from other nodes. To test out network partition:
+```console
+$ ./drop-input.sh
+```
+
+Accept packets from other nodes. Only works if drop-input.sh was run before:
```

Comment:
I'd call this `reset-input` or `reset-network` - why cannot it be a no-op if `drop-input` has been invoked prior?